sandbox/src/components/MessageFeedDemo/MessageFeed.js

```diff
@@ -0,0 +1,628 @@
+                                  surface:
+                                    "mobileapp://com.adobe.sampleApp/feed/promos"
+                                },
+                                content: {
+                                  imageUrl:
+                                    "https://media.giphy.com/media/ADgfsbHcS62Jy/giphy.gif",
+                                  actionTitle: "Shop the sale!",
+                                  actionUrl: "https://luma.com/sale",
+                                  body: "Now you're ready to earn!",
+                                  title: "Funds deposited!"
+                                },
+                                contentType: "application/json"
+                              },
+                              id: "0263e171-fa32-4c7a-9611-36b28137a81d"
+                            },
+                            id: "0263e171-fa32-4c7a-9611-36b28137a81d"
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                }
+              ],
+              scope: "web://target.jasonwaters.dev/aep.html"
+            }
+          ],
+          type: "personalization:decisions",
+          eventIndex: 0
+        },
+        {
+          payload: [
+            {
+              scope: "Target",
+              hint: "35",
+              ttlSeconds: 1800
+            },
+            {
+              scope: "AAM",
+              hint: "9",
+              ttlSeconds: 1800
+            },
+            {
+              scope: "EdgeNetwork",
+              hint: "or2",
+              ttlSeconds: 1800
+            }
+          ],
+          type: "locationHint:result"
+        },
+        {
+          payload: [
+            {
+              key: "kndctr_4DA0571C5FDC4BF70A495FC2_AdobeOrg_cluster",
+              value: "or2",
+              maxAge: 1800,
+              attrs: {
+                SameSite: "None"
+              }
+            }
+          ],
+          type: "state:store"
+        }
+      ]
+    };
+    async function applyRuleEngineResponse() {
+      window.alloy("applyResponse", {
+        renderDecisions: true,
+        decisionContext: {
+          color: "pink",
+          action: "lipstick"
+        },
+        responseBody: mockResponse
+      });
+    }
+    document
+      .getElementById("social-media-share")
+      .addEventListener("click", evt => {
+        evt.stopImmediatePropagation();
+        evt.preventDefault();
+
+        window.alloy("evaluateRulesets", {
+          action: "share-social-media"
+        });
+      });
+
+    document.getElementById("deposit-funds").addEventListener("click", evt => {
+      evt.stopImmediatePropagation();
+      evt.preventDefault();
+
+      window.alloy("evaluateRulesets", {
+        action: "deposit-funds"
+      });
+    });
+
+    document.getElementById("magic").addEventListener("click", evt => {
```

Comment:
Recommend to remove any code that isn't relevant to make the sample work, like this click handler for `magic`